README.md

```diff
@@ -27,20 +27,29 @@ Dependencies in Debian trixie:
 build-essential meson ninja-build pkg-config qt5-qmake qttools5-dev-tools qtbase5-dev libtermpaint-dev libtuiwidgets-dev libposixsignalmanager-dev
 ```
 
+Installing dependencies in Fedora:
+```
+sudo dnf install gcc gcc-c++ meson ninja-build pkg-config qt5-qtbase-devel qt5-linguist
+```
+
+Fedora do not have packages for [Tui Widgets](https://tuiwidgets.namepad.de/) and [Termpaint](https://termpaint.namepad.de/) so you have to install them from sources.
+
 For third-party syntex highlighting you need the compile
 option: `-Dsyntax_highlighting=true` and the following additional dependencies:
 ```
 libkf5syntaxhighlighting-dev libkf5syntaxhighlighting-tools cmake
 ```
 
+or in Fedora:
 
-See also [Tui Widgets](https://tuiwidgets.namepad.de/)
-and [Termpaint](https://termpaint.namepad.de/)
```

Comment:
These links should stay here. If we want to avoid repeating them they should be removed from the fedora specific parts.